### PR TITLE
Bump version to 0.9

### PR DESCRIPTION
**Pull request by randombit**

### CODE REVIEW

**Comment by altkdf** [comment]
Should we also add the changes to the release notes?

File: RELEASES.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Unreleased
 
+# 0.9.0
+## Changed
+- Smaller stack usage during final exponentiation
+- Serialization of Gt elements
```
Comment (altkdf) [inline RELEASES.md:6]: From the text it sounds to me like we implemented the serialization recently, but I think it's about the change in the endianness?

Comment (altkdf) [reply RELEASES.md:6]: So mainstream changes?

Comment (randombit) [reply RELEASES.md]: All of this file was just from the upstream. I added some notes providing the changes in this fork

**Review by altkdf — APPROVED** [review]

File: RELEASES.md (hunk 1 above)
Comment (altkdf) [reply RELEASES.md]: thanks!

**Pull request merged**